main/src/library/Set.flix

```diff
@@ -517,9 +517,18 @@ namespace Set {
     /// Returns `s` as a mutable set.
     ///
     @Time(1) @Space(1)
-    pub def toMutSet(s: Set[a]): MutSet[a] & Impure =
+    pub def toMutSet(s: Set[a]): MutSet[a, Impure] & Impure =
         MutSet(ref s)
 
+    ///
+    /// Returns `s` as a mutable set given the region `r`.
+    ///
+    @Time(1) @Space(1)
+    pub def toMutSetWithRegion(r: Region[r], s: Set[a]): MutSet[a, r] & reg(r) with Order[a] =
```

Comment:
Does anything depend on this? Can we remove it?

Reply:
I added this to experiment with converting a `Set` to `MutSet` using regions, but I can easily remove it as it has no dependencies